UnRen.bat

```diff
@@ -1,115 +1,114 @@
 @echo off
-REM [DEV//WARN_START
+:: [DEV//WARN_START
 title UnRen.bat Error
-echo.
+echo/
 echo   Error!
-echo.
+echo/
 echo   This is the development version of UnRen. This is non functional and not
 echo   intended to be used by end users.
 echo   Please use the release version instead, found in the F95zone thread:
-echo.
+echo/
 echo   https://f95zone.to/threads/unren-bat-rpa-extractor-rpyc-decompiler-console-developer-menu-enabler.3083/
-echo.
+echo/
 pause>nul|set/p=.  Press any key to exit...
-echo.
+echo/
 pause
 exit
-REM DEV//WARN_END]
-REM --------------------------------------------------------------------------------
-REM Configuration:
-REM   Set a Quick Save and Quick Load hotkey - http://www.pygame.org/docs/ref/key.html
-REM --------------------------------------------------------------------------------
+:: DEV//WARN_END]
+:: --------------------------------------------------------------------------------
+:: Configuration:
+::   Set a Quick Save and Quick Load hotkey - http://www.pygame.org/docs/ref/key.html
+:: --------------------------------------------------------------------------------
 set "quicksavekey=K_F5"
 set "quickloadkey=K_F9"
-REM --------------------------------------------------------------------------------
-REM !! END CONFIG !!
-REM --------------------------------------------------------------------------------
-REM The following variables are Base64 encoded strings for unrpyc and rpatool
-REM Due to batch limitations on variable lengths, they need to be split into
-REM multiple variables, and joined later using powershell.
-REM --------------------------------------------------------------------------------
-REM unrpyc by CensoredUsername
-REM   https://github.com/CensoredUsername/unrpyc
-REM Edited to remove multiprocessing and adjust output spacing [UNRPYC//SHA] [UNRPYC//DATE]
-REM   https://github.com/F95Sam/unrpyc
-REM --------------------------------------------------------------------------------
-REM set unrpyccab01=
+:: --------------------------------------------------------------------------------
+:: !! END CONFIG !!
+:: --------------------------------------------------------------------------------
+:: The following variables are Base64 encoded strings for unrpyc and rpatool
+:: Due to batch limitations on variable lengths, they need to be split into
+:: multiple variables, and joined later using powershell.
+:: --------------------------------------------------------------------------------
+:: unrpyc by CensoredUsername
+::   https://github.com/CensoredUsername/unrpyc
+:: Edited to ::ove multiprocessing and adjust output spacing [UNRPYC//SHA] [UNRPYC//DATE]

```

Comment:
mistaken find/replace with "Remove" -> "::ove"

Reply:
Did'n know about the problems with :: for replacing REM. Interesting.
Reverted completely to REM.